pkg/apis/externaldns/types.go

```diff
@@ -51,6 +54,11 @@ func NewConfig() *Config {
 
 // ParseFlags adds and parses flags from command line
 func (cfg *Config) ParseFlags(args []string) error {
+	defaultDryRun, err := strconv.ParseBool(os.Getenv("DRY_RUN"))
+	if err != nil {
+		defaultDryRun = true
```

Comment:
I think I'll do that. Env var processing is not high prio atm. I'll leave this open till then. Molecule also requires ALIAS and multi-zone. So no need to rush this here.